### PR TITLE
feat(mainlayout): add navbar connected

### PR DESCRIPTION
**Pull request by Anto2441**
## Description

⚠️ The purpose of this component is not to remain but to present the elements to be displayed in the navigation bar when the user's state is managed.

- Add a new **`NavBarConnected`** component which displays new content when the user is logged in
- Add his stories
- Add **`NavBar`** stories

Capture d’écran 2024-03-25 à 12 38 27

### CODE REVIEW

File: src/shared/ui/layouts/MainLayout/NavBarConnected.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+import { CiSettings } from "react-icons/ci";
+import { GoBell } from "react-icons/go";
+import { LuArrowDownCircle } from "react-icons/lu";
+import { IconButton } from "../../inputs";
+import { buttonVariants } from "../../inputs/Button/Button";
+import { Link } from "../../navigation";
+
+export function NavBarConnected() {
+  return (
+    <div className="bg-color-watermark flex items-center justify-between px-spacing-base">
```
Comment (carlo-ferreira) [inline src/shared/ui/layouts/MainLayout/NavBarConnected.tsx]: Where does `bg-color-watermark` come from ^^?

File: src/shared/ui/layouts/MainLayout/NavBarConnected.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+            </Link>
+          </li>
+          <li>
+            <Link href="/">
+              <IconButton size="small" tooltip="news" variant="over-media">
+                <GoBell />
+              </IconButton>
+            </Link>
+          </li>
+          {/* TODO: replace with DropdownMenu component when it's ready */}
```
Comment (carlo-ferreira) [inline src/shared/ui/layouts/MainLayout/NavBarConnected.tsx:68]: 👌 👌 👌

**Review by carlo-ferreira — APPROVED** [review]
Thank you for the `stories` files!

I left a small comment about a background color. Do we need it? Can we remove it may be? I'm approving the PR :)

File: src/shared/ui/layouts/MainLayout/NavBarConnected.tsx (hunk 1 above)
Comment (Anto2441) [reply src/shared/ui/layouts/MainLayout/NavBarConnected.tsx]: [Travolta](https://media3.giphy.com/media/v1.Y2lkPTc5MGI3NjExMWVxMWUydmxpbTEweWoyd292aTBqbW1sNjNzcmhhdnA1NG84enJkcCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/2vs70gBAfQXvOOYsBI/giphy.gif)

Remove with a `git commit --amend` 😬

**Pull request merged**